nextroute/schema/input.go

```diff
@@ -101,3 +102,10 @@ type Location struct {
 	Lon float64 `json:"lon"`
 	Lat float64 `json:"lat"`
 }
+
+// DurationGroup represents a group of stops that get additional process time
+// (duration) whenever a stop of the group is approached for the first time.
+type DurationGroup struct {
+	Group    []string `json:"group,omitempty"`
```

Comment:
If we do that we need to name the field in the input json also stops. I feel that this is confusing, since it is a duration group and they are not duration stops. So I think, while may seem repetitive in the code, it is still the better option when we look at the user side of things.